doc/design.md

```diff
@@ -78,45 +85,50 @@ Documents schema
-- Linux and BSD file systems are sensible to the case, OSX and Windows are not
-  (they preserve the original case, but they consider `foo` and `FOO` to be
-  the same file).
+- The path separator is `/` everywhere, except on Windows where it's
+  `\ `.
+- Linux and BSD file systems are sensible to the case, OSX and Windows
+  are not (they preserve the original case, but they consider `foo` and
+  `FOO` to be the same file).
 - Linux and BSD use UTF-8 encoding for filenames, with no normalization.
-  Windows uses UTF-16. OSX does something stupid: UTF-8 with a normalization
-  that is nearly the unicode NFD, but not exactly.
-- `/` and the NULL character are forbidden on all the OSes. On Windows, the
-  list is longer: `"/\*?<>|:`.
+  Windows uses UTF-16. OSX does something stupid: UTF-8 with a
+  normalization that is nearly the unicode NFD, but not exactly.
+- `/` and the NULL character are forbidden on all the OSes. On Windows,
+  the list is longer: `"/\*?<>|:`.
 - They are a bunch more restrictions on Windows:
   - the length of a path is limited to 260 characters
   - some names are reserved, like `AUX`, `COM1` or `LPT1`
   - a file or directory name cannot end with a space or a period.
 
-Node.js helps us a bit, but we are on our own for most of the things in this
-list.
+Node.js helps us a bit, but we are on our own for most of the things in
+this list.
 
-For detecting conflicts, we need to know if two paths are the same. With the
-issue of case sensitivity and NFD normalization, it's not as easy as it seems.
-For Linux and BSD, we take the path and we put it in the `_id` field. For
-Windows and OSX, we make the path upper case before putting it in `_id` field.
-For OSX, we also does a string normalization on this field. Now, when a new
-file is added, we can check in pouchdb and see if another file has path that
-will collide with this new path.
+For detecting conflicts, we need to know if two paths are the same. With
+the issue of case sensitivity and NFD normalization, it's not as easy as
+it seems. For Linux and BSD, we take the path and we put it in the `_id`
+field. For Windows and OSX, we make the path upper case before putting
+it in `_id` field. For OSX, we also does a string normalization on this
+field. Now, when a new file is added, we can check in pouchdb and see if
+another file has path that will collide with this new path.
 
 So, even if `path` and `_id` are very similar, they have distinct roles:
 
 - `_id` is the normalized form and is used for comparison of paths
-- `path` is the prefered form and used for actions on local file system and
-  remote cozy.
+- `path` is the prefered form and used for actions on local file system
+  and remote cozy.
 
-The permissions are different on Unix and on Windows. So, the `executable`
-field is just ignored on Windows.
+The permissions are different on Unix and on Windows. So, the
+`executable` field is just ignored on Windows.
+
+
+[0]:  workflow.png
```

Comment:
It looks weird to use `[0]` on this file. There is only one image and the line with it was short.